specification/billing/resource-manager/readme.md

```diff
@@ -28,6 +28,14 @@ These are the global settings for the Billing API.
 openapi-type: arm
 tag: package-2020-05
 ```
+### Tag: package-2020-11-privatepreview
```

Comment:
How are you ensuring that the private preview version is not visible to public users?